google/google-how-to/gce/set-hostname-cloudinit.rst

```diff
@@ -0,0 +1,49 @@
+Set hostname with cloud-init on GCE
+===================================
+
+Starting in 24.04 GCE images, cloud-init is configured with no ``/etc/hostname`` file and the cloud-init key ``create_hostname_file`` set to false.  As such, setting the instance hostname using solely user-supplied metadata will be overwritten by Google's DHCP service.
+
+
+To set the hostname using DHCP
+------------------------------
+
+To align the DHCP-provided hostname with the user-data provided hostname, follow Google's documentation to `Create a VM instance with a custom hostname`_
```

Comment:
What does 'align' mean in this context? Could you give a bit of an explanation, maybe with an example? 
Also, it would help to know why we might need to set the hostname using DHCP (I thought Google does that automatically?).